sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1361,87 +1369,18 @@ def __init__(
     self.triggering_frequency = triggering_frequency
     self.insert_retry_strategy = insert_retry_strategy
     self._validate = validate
+    self._temp_file_format = temp_file_format or bigquery_tools.FileFormat.JSON
```

Comment:
AFAICT using Avro has no disadvantages compared to JSON for loading data into BigQuery, but would requiring a schema constitute a breaking API change for semantic versioning purposes?

Personally I'm for using Avro as default. I guess when users update Beam, they'll specify a `temp_file_format` explicitly to get the old behavior.

Reply:
Since this is technically still experimental, and masked behind a flag, I think it makes sense to make avro the main way of doing it (and simply add a check that the schema was passed). cc: @chamikaramj

Reply:
Oh I didn't realize it was experimental. I'll make the change then!

Reply:
+1 for making Avro the default for the new sink.